intera_interface/src/intera_motion_interface/motion_trajectory.py

```diff
@@ -70,14 +70,22 @@ def __init__(self, label = None,
     def stop_trajectory(self):
         self._client.stop_trajectory()
 
-    def send_trajectory(self):
+    def send_trajectory(self, wait_for_result=True):
         if not self._traj.waypoints:
             rospy.logerr("Trajectory is empty! Cannot send.")
             return None
         self._check_options()
         self._client.send_trajectory(self.to_msg())
-        result = self._client.wait_for_result()
-        return result
+
+        if wait_for_result:
```

Comment:
I agree - in this case it is more clear to explicitly return none, rather than to rely on this happening implicitly.